src/client.ts

```diff
@@ -1591,6 +1592,33 @@ export class StreamChat<
     });
   }
 
+  /**
+   * getRateLimits - Returns the rate limits quota and usage for the current app, possibly filter for a specific platform and/or endpoints.
+   * Only available server-side.
+   *
+   * @param {object} params The params for the call. If none of the params are set, all limits for all platforms are returned.
+   */
+  async getRateLimits(params?: {
+    android?: boolean;
+    endpoints?: string[];
```

Comment:
we need to type all the different endpoint names that we support and keep them in sync here, it can also be reused on the response type